src/config/client_config.cc

```diff
@@ -53,7 +48,7 @@ ClientConfig::ClientConfig(int flags, std::string_view ip, std::string_view user
     auto sIP = ip.empty() ? "" : fmt::format(" IP {}", ip);
     auto sUA = userAgent.empty() ? "" : fmt::format(" UserAgent {}", userAgent);
     cInfo.name = fmt::format("Manual Setup for{}{}", sIP, sUA);
-    clientInfo = std::make_unique<ClientInfo>(std::move(cInfo));
+    clientInfo = std::move(cInfo);
```

Comment:
unneeded?

Reply:
Probably. I remember doing this to avoid some compiler or clang-tidy warning. Probably not relevant now. I'll revert.